Assets/HoloToolkit-Examples/ARCA/Scripts/ColorChanger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
```

Comment:
classes should be in a HoloToolkit namespace. I recommend using HoloToolkit.ARCA or HoloToolkit.AugmentedRealityCapture

Reply:
I'm in favor of HoloToolkit.AugmentedRealityCapture

Reply:
We will change the namespace to HoloToolkit.ARCapture, simple, descriptive and memorable.